Mimir/Controllers/AvatarController.cs

```diff
@@ -1,23 +1,136 @@
+using System.Numerics;
+using Bencodex;
+using Libplanet.Common;
+using Libplanet.Crypto;
 using Microsoft.AspNetCore.Mvc;
 using Mimir.Models.Avatar;
 using Mimir.Repositories;
+using Mimir.Services;
+using Mimir.Util;
+using MongoDB.Bson;
+using Nekoyume.Model.State;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace Mimir.Controllers;
 
 [ApiController]
 [Route("{network}/avatars")]
 public class AvatarController(AvatarRepository avatarRepository) : ControllerBase
 {
+    #region temporary snippets
+
+    // This is a snippet from Mimir/Util/BigIntegerToStringConverter.cs
```

Comment:
It may be better to create a new csharp project like `Mimir.Shared` to share common code.

Reply:
Yeah, that's what I was thinking too, maybe we could do [a follow-up issue](https://github.com/planetarium/mimir/issues/52).